lightning/restore/restore.go

```diff
@@ -1426,6 +1432,13 @@ func (cr *chunkRestore) deliverLoop(
 	var channelClosed bool
 	var dataKVs, indexKVs []kvenc.KvPair
 
+	deliverTask := t.logger.With(
+		zap.Int32("engineNumber", engineID),
+		zap.Int("fileIndex", cr.index),
+		zap.Stringer("path", &cr.chunk.Key),
+		zap.String("task", "deliver"),
+	).Begin(zap.InfoLevel, "deliver started")
```

Comment:
What's this?

Reply:
Logging detail information if the write to engine error occurred.

Reply:
You didn't call `deliverTask.End()` though, so using the result of `With()` as a logger is sufficient.

Reply:
addressed